redbot/cogs/mod/kickban.py

```diff
@@ -214,6 +225,19 @@ class KickBanMixin(MixinMeta):
         If days is not a number, it's treated as the first word of the reason.
         Minimum 0 days, maximum 7. Defaults to 0."""
 
+        author = ctx.author
+        guild = ctx.guild
+        if reason is None:
+            reason = "No reason was given."
```

Comment:
This should be wrapped in `_()` for localization.
```suggestion
            reason = _("No reason was given.")
```